setup.py

```diff
@@ -8,7 +8,7 @@
 include_path = [np.get_include()]
 
 ext_modules = cythonize('pyembree/*.pyx', language='c++',
-                        include_dirs=include_path)
+                        include_path=include_path)
```

Comment:
It seems as though `include_path` has been broken in some very recent (0.24) versions of Cython. I would rather we stay away from it.  Was this line causing you problems for some reason?

Reply:
Well, I did not have Cython. I installed it with pip, which gave me the lastest version( 0.27.3). With this version, I had the following error : `ValueError: got unknown compilation option, please remove: include_dirs`

With CI, we have the same error, as you can see: https://github.com/scopatz/pyembree/pull/10/commits/57199d98346bbeb0f3ea5671fc04e19ef3954417

IMHO, I think you can accept it.

Reply:
Ahh yeah, I see the below now.